analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/IssueLocationCollectorTest.cs

```diff
@@ -32,7 +32,7 @@ public partial class IssueLocationCollectorTest
         private static TextLine GetLine(int lineNumber, string code) =>
             SourceText.From(code).Lines[lineNumber];
 
-        private static void VerifyIssueLocations(IEnumerable<IIssueLocation> result,
+        private static void VerifyIssueLocations(IReadOnlyCollection<IIssueLocation> result,
             IEnumerable<bool> expectedIsPrimary,
```

Comment:
nitpick: The method parameters lines 36-38 should be aligned with the one on line 35.